### PR TITLE
CASMINST-4390 1.2.5 : DOCS: K8s upgrade on w001 throws error

### PR DESCRIPTION
**Pull request by kimjensen-hpe**
## Summary and Scope

Add to postgres troubleshooting to cover the case where reinit fails and leader pod needs to be restarted.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMINST-4390](DOCS: K8s upgrade on w001 throws error)
* Change will also be needed in NA
* Future work required by NA
* Documentation changes required in NA
* Merge with/before/after NA

## Testing

This issue was seen on drax where 0.1.11 to 1.2.0 beta 96 upgrade was occurring.
In the past this procedure has been used instead of waiting 45m for the lag to resolve.

## Risks and Mitigations

Low

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [x] Target branch correct
- [ ] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by bklei — APPROVED** [review]

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 1]
```diff
@@ -186,105 +186,154 @@ Example output:
...
+    ```
 
-If the check above finds such files, first find any zero length pg_internal.init.* files.
-```bash
-postgres@keycloak-postgres-2:~$ find /home/postgres/pgdata -name pg_internal.init.* -size 0 
-```
+    Example output:
 
-Example output:
+    ```
```
Comment (cdelatte-hpe) [inline operations/kubernetes/Troubleshoot_Postgres_Database.md:246]:
```suggestion
    ```text
```

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 2]
```diff
@@ -186,105 +186,154 @@ Example output:
...
+    ```bash
+    ncn-w001# kubectl exec keycloak-postgres-2 -n services -it -- bash
+    postgres@keycloak-postgres-2:~$ export LOG=`ls -t /home/postgres/pgdata/pgroot/pg_log/*.csv | head -1`
+    postgres@keycloak-postgres-2:~$ grep pg_internal.init $LOG | grep "invalid segment number 0" | tail -1
+    ```
 
-Then delete the zero length pg_internal.init.* files. Double check the syntax of the command in this step before executing it `-size 0 -exec rm {} \;`.
+    Example output:
 
+    ```
```
Comment (cdelatte-hpe) [inline operations/kubernetes/Troubleshoot_Postgres_Database.md:266]:
```suggestion
    ```text
```

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 3]
```diff
@@ -186,105 +186,154 @@ Example output:
...
+    postgres@keycloak-postgres-2:~$ find /home/postgres/pgdata -name pg_internal.init.* -size 0 
+    ```
 
-```bash
-postgres@keycloak-postgres-2:~$ grep pg_internal.init $LOG | grep "invalid segment number 0" | tail -1
-```
+    Example output:
 
-Example output:
+    ```
```
Comment (cdelatte-hpe) [inline operations/kubernetes/Troubleshoot_Postgres_Database.md:278]:
```suggestion
    ```text
```

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 4]
```diff
@@ -186,105 +186,154 @@ Example output:
...
+        ncn-w001# kubectl delete pod cray-console-data-postgres-0 -n services
+        ```
+    
+        ```bash
+        ncn-w001# kubectl exec keycloak-postgres-0 -c postgres -n services -it -- patronictl list
+        ```
+    
+        Example output:
+    
+        ```
```
[the remark this answers is not in the document]
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

File: operations/kubernetes/Troubleshoot_Postgres_Database.md (hunk 1 above)
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 5]
```diff
@@ -186,105 +186,154 @@ Example output:
...
+    postgres@keycloak-postgres-2:~$ find ~/pgdata -name pg_internal.init.2239188
+    ```
 
-```bash
-ncn-w001# kubectl exec keycloak-postgres-0 -c postgres -n services -it -- patronictl list
-```
+    Example output:
 
-Example output:
+    ```
```
[the remark this answers is not in the document]
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 6]
```diff
@@ -186,105 +186,154 @@ Example output:
...
-+-------------------+---------------------+--------------+--------+---------+----+-----------+
-```
+    ```bash
+    ncn-w001# kubectl exec cray-console-data-postgres-0 -n services -- bash
+    root@cray-console-data-postgres-0:~# patronictl reinit cray-console-data-postgres cray-console-data-postgres-1
+    ```
+
+    Example output:
+
+    ```bash
```
[the remark this answers is not in the document]
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 7]
```diff
@@ -186,105 +186,154 @@ Example output:
...
+
+    Iterate over the above steps to find, locate and delete non-zero length pg_internal.init.* files until there are no more new `invalid segment number 0` messages. At this point, verify that the cluster member has started.
+
+    ```bash
+    ncn-w001# kubectl exec keycloak-postgres-0 -c postgres -n services -it -- patronictl list
+    ```
+
+    Example output:
+
+    ```
```
[the remark this answers is not in the document]
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

File: operations/kubernetes/Troubleshoot_Postgres_Database.md (hunk 2 above)
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

**Review by gbaker-hpe — APPROVED** [review]

**Pull request merged**